src/main/java/internet/shop/db/Storage.java

```diff
@@ -0,0 +1,55 @@
+package internet.shop.db;
```

Comment:
I think we should use `Storage` for storing data solely but not to perform operations inside of it, except of `addProduct`
My reference was - `Use Storage class as a persistence layer.` in hw description, see pic below
![Screenshot 2021-01-11 at 18 33 59](https://user-images.githubusercontent.com/67925366/104210587-bd42b000-543b-11eb-9b45-fc1dd339ce92.png)